zap/src/main/java/org/parosproxy/paros/extension/ExtensionLoader.java

```diff
@@ -862,7 +864,8 @@ private void hookAllExtension(double progressFactor) {
         for (int i = 0; i < getExtensionCount(); i++) {
             final Extension ext = getExtension(i);
             try {
-                logger.info("Initializing " + ext.getDescription());
+                String name = ext.getUIName() == null ? ext.getName() : ext.getUIName();
```

Comment:
Have you encountered a case where the UI name is null? The contract is that it's never null (it defaults to the name also).

Reply:
Nope, was just trying to be defensive about it. I’ll adjust the code.